### PR TITLE
Update ZipJs from 2.4.0/2.5.0 to 3.1.5

### PR DESCRIPTION
**Pull request by kestred**
Includes changes as requested by @calvinmetcalf in https://github.com/calvinmetcalf/shapefile-js/pull/91.

The original fork the Pull Request was made from had been archived, so I've made a new PR.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -59,52 +59,53 @@ shp.combine = function(arr) {
   return out;
 };
 shp.parseZip = function(buffer, whiteList) {
-  var key;
+  var key, i;
```
Comment (calvinmetcalf) [inline lib/index.js]: don't need i here, otherwise good

**Comment by kestred** [comment]
👌

**Comment by kestred** [comment]
Bump.

**Comment by calvinmetcalf** [comment]
sorry will take a look latter today

**Comment by jmunowitch** [comment]
I hate to do this, but I must ask. Is there any plans to take a look at this PR?

If not, I can create a fork with this change and use that.

**Comment by calvinmetcalf** [comment]
done

**Pull request closed without merging**